sparkbox/apprentices-page-next
# feat: add current apprentices

## Description

- Created tests for current-apprentices component
- Built current-apprentices component with prop types
- Used getStaticProps in index.js to pass in our data 
- Added Styles to current-appretnices component

### Spec

Designs: [Link to design](https://www.figma.com/file/eeSQQIRz0R5Ag9SIJJH6Hx/apprentices.sparkbox.com?node-id=0%3A1)

See Story: [FSA21V2-148](https://sparkbox.atlassian.net/browse/FSA21V2-148)

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Pull down all related branches.
3. Confirm all tests pass: `npm run test:ci`
4. run npm lint to confirm linter passes

## Validation

The following has been completed by the developer:

_[delete anything irrelevant to this PR]_


### Browser Testing

_[delete if irrelevant to this issue]_

#### Gold Level Browsers

In these browsers, all content is accessible and design matches exactly. In most cases, bugs should be resolved before merge.

**macOS**


**Windows**


**Mobile**


#### Silver Level Browsers

In these browsers, all content is readable, but design may be inaccurate.

**Windows**


**Mobile**


#### Bronze Level Browsers

In these browsers, our only goal is content readability.

**Windows**
